bfdd/ptm_adapter.c

```diff
@@ -153,7 +153,7 @@ static int _ptm_msg_address(struct stream *msg, int family, const void *addr)
 	return 0;
 }
 
-int ptm_bfd_notify(struct bfd_session *bs)
+int ptm_bfd_notify(struct bfd_session *bs, uint8_t notify_state)
```

Comment:
Please, don't change this function signature.... Again you are using information that already exists in `struct bfd_session`.